### PR TITLE
Update benchmarks in the README

### PR DESCRIPTION
**Pull request by ruiaraujo**
There was significant improvements since the last version.

### CODE REVIEW

**Comment by ruiaraujo** [comment]
:+1:

**Comment by angelaigreja** [comment]
:+1:

**Comment by ruiaraujo** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -11,15 +11,20 @@ code which yields much better performance.
...
-graphql-jit x 123,367 ops/sec ±1.13% (81 runs sampled)
-Fastest is graphql-jit
-
-$ ts-node -T ./src/__benchmarks__/schema-many-resolvers.benchmark.ts 
-graphql-js x 14,006 ops/sec ±2.05% (78 runs sampled)
-graphql-jit x 52,248 ops/sec ±1.36% (81 runs sampled)
-Fastest is graphql-jit
+$ ts-node -T ./src/__benchmarks__/benchmarks.ts skip-json
+Starting introspection
+graphql-js x 1,155 ops/sec ±1.55% (215 runs sampled)
```
Comment (vigneshshanmugam) [inline README.md]: the number difference seems drastic, was the current run significantly differ so much from the previous one?

Comment (ruiaraujo) [reply README.md]: The readme only used to include the `fewResolvers` and `manyResolvers`.

**Pull request merged**